tests/Lykke.Payments.Link4Pay.Tests/Lykke.Payments.Link4Pay.Tests.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
downgrade?

Reply:
it's not easy to use .net core 3.1 with current infrastructure packages